packages/atomic/src/pages/examples/commerce-website/search.html

```diff
@@ -29,8 +29,13 @@ <h1>Search page</h1>
         <atomic-commerce-interface type="search" style="max-width: 1000px; margin: auto">
           <atomic-search-layout>
             <atomic-layout-section section="search">
-              <atomic-commerce-search-box textarea></atomic-commerce-search-box>
-              <!-- <atomic-commerce-search-box redirection-url="https://docs.coveo.com/en/1458" textarea></atomic-commerce-search-box> -->
+              <atomic-commerce-search-box textarea>
```

Comment:
Unrelated to the PR: `textarea` property is not doing anything/atomic commerce search box is already a text area by default.

Reply:
```suggestion
              <atomic-commerce-search-box>
```
quickwin